src/models/stores/unit-configuration.ts

```diff
@@ -79,4 +79,8 @@ export interface UnitConfiguration extends ProblemConfiguration {
   comparisonPlaceholderContent: string | string[];
   // configuration of navigation tabs (document navigation UI)
   navTabs: SnapshotIn<typeof NavTabsConfigModel>;
+  // used for AI tagging
+  showCommentTag?: boolean;
+  defaultTag?: string;
+  commentTags?: Record<string, string> | never[];
```

Comment:
```suggestion
  commentTags?: Record<string, string>;
```
The `| never[]` is not necessary -- I checked.